docs/components.md

```diff
@@ -0,0 +1,41 @@
+# Components
+
+The distribution offers support for the following components.
+
+## Beta
+
+These components are considered stable. While in beta, breaking changes may be
+introduced in a new release. In addition, any of these components may be
+removed prior to the 1.0 release.
+
+| Receivers        | Processors        | Exporters | Extensions    |
+| :--------------: | :--------:        | :-------: | :--------:    |
+| fluentforward    | attributes        | file      | healthcheck   |
+| hostmetrics      | batch             | logging   | httpforwarder |
+| jaeger           | filter            | otlp      | host_observer |
+| k8s_cluster      | k8s_tagger        | sapm      | k8s_observer  |
+| kubeletstats     | memorylimiter     | signalfx  | pprof         |
+| opencensus       | metrictransform   | splunkhec | zpages        |
+| otlp             | resource          |           |               |
+| receiver_creator | resourcedetection |           |               |
+| sapm             | span              |           |               |
+| signalfx         |                   |           |               |
+| simpleprometheus |                   |           |               |
+| smartagent       |                   |           |               |
+| splunkhec        |                   |           |               |
```

Comment:
If we going by the config names of components in this table, `splunkhec` -> `splunk_hec`.

Reply:
Fixed!